### PR TITLE
fix: use setBaseUrl when constructor

### PR DESCRIPTION
**Pull request by woomurf**
- baseUrl defined with default value and use setBaseUrl in constructor

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: src/ainftBase.ts [hunk 1]
```diff
@@ -13,8 +13,9 @@ export default class AinftBase {
     route?: string,
   ) {
     this.route = route || '';
-    this.baseUrl = baseUrl + this.route;
     this.ain = ain;
+
+    this.setBaseUrl(baseUrl);
```
Comment (platfowner) [inline src/ainftBase.ts:18]: ah, what about making baseUrl optional as it already has a default value?

Comment (woomurf) [reply src/ainftBase.ts]: Done!

Comment (platfowner) [reply src/ainftBase.ts]: 👍

**Comment by woomurf** [comment]
Thanks for review! now merged!

**Pull request merged**